stdlib/public/libexec/swift-backtrace/main.swift

```diff
@@ -202,6 +203,10 @@ Generate a backtrace for the parent process.
 --crashinfo <addr>
 -a <addr>               Provide a pointer to a platform specific CrashInfo
                         structure.  <addr> should be in hexadecimal.
+
+--symbolicate [<bool>]  Set whether or not to resolve addresses into symbols.
+                        Also controls whether or not we look for inline frame
```

Comment:
This is a bit vague about what that control looks like, although it's fairly obvious. Maybe say something like, "When symbolication is disabled, inline frame data is also not used and the backtrace only shows the actual stack frame return addresses." Except better, because I don't know how to word it well.

Reply:
I've updated the text, but that's partly because I added another option :-)

Reply:
Looks good!